aws/common_columns.go

```diff
@@ -146,3 +146,23 @@ func getCallerIdentity(ctx context.Context, d *plugin.QueryData, _ *plugin.Hydra
 	d.ConnectionManager.Cache.Set(cacheKey, callerIdentity)
 	return callerIdentity, nil
 }
+
+func getAccountPartition(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	cacheKey := "getAccountPartition"
```

Comment:
We already cache the caller identity (which is called from `getCommonColumns` and is probably the only API call we'd do here), is there any significant benefit to caching this partition value?